src/Algebra/Construct/Pointwise.agda

```diff
@@ -0,0 +1,224 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- For each `IsX` algebraic structure `S`, lift the structure to the
+-- 'pointwise' function space `A → S`: categorically, this is the
```

Comment:
(Definitely a comment, not a 'todo'): I wonder if it wouldn't be just as easy to do the n-ary version, i.e. lift to n arguments on the left. Without going through currying/uncurrying, but via the `Nary` facilities already in the library.

Reply:
Oooh... interesting!? (Always more to have to get my head around... ;-))

Reply:
Have concluded in the interests of time/brain that this be a downstream consideration.